packages/react/package.json

```diff
@@ -17,9 +17,24 @@
     "umd/",
     "jsx-runtime.js",
     "jsx-dev-runtime.js",
+    "unstable-index.server.js",
     "unstable-cache.js"
   ],
   "main": "index.js",
+  "exports": {
+    ".": {
+      "react-server": "./unstable-index.server.js",
+      "default": "./index.js"
+    },
+    "./index": {
+      "react-server": "./unstable-index.server.js",
+      "default": "./index.js"
+    },
+    "./build-info.json": "./build-info.json",
+    "./jsx-runtime": "./jsx-runtime.js",
+    "./jsx-dev-runtime": "./jsx-dev-runtime.js",
+    "./": "./"
```

Comment:
The exports protocol is an allow list so we must list everything we want to expose here. That's actually great because we can explicitly forbid deep linking. Unfortunately our build tooling is using this same file to find things so  we need to export everything until we fix that.